### ISSUE TITLE
zart 2.7.5 doesn't build

### ISSUE THREAD

**Issue by antonio-rojas** [c-koi/zart#13]
```
src/ImageConverter.cpp: In static member function ‘static void ImageConverter::convert(const cimg_library::CImg<float>&, QImage*)’:
src/ImageConverter.cpp:117:27: error: ‘const struct cimg_library::CImg<float>’ has no member named ‘spectrum’; did you mean ‘_spectrum’?
  117 |   const int spectrum = in.spectrum();
      |                           ^~~~~~~~
      |                           _spectrum
src/ImageConverter.cpp:119:27: error: ‘const struct cimg_library::CImg<float>’ has no member named ‘data’; did you mean ‘_data’?
  119 |   const float * srcR = in.data(0, 0, 0, 0);
      |                           ^~~~
      |                           _data
```

**Comment by kuboosoft** [comment]
Same issue, Zart does't build...

**Comment by c-koi** [comment]
Should be fixed by 2d8727b5dc6d6783a71a0f161112ec588d6b17ad

**Comment by antonio-rojas** [comment]
The commit doesn't apply on top of 2.7.5, but latest git head works fine.
